opi4/kernel.config

```diff
@@ -0,0 +1,5 @@
+CONFIG_TYPEC=y
+CONFIG_DRM=y
+CONFIG_DRM_ROCKCHIP=y
+CONFIG_TYPEC_FUSB302=y
+CONFIG_TYPEC_TCPM=y
```

Comment:
I believe we do not need CONFIG_TYPEC, CONFIG_TYPEC_FUSB302, CONFIG_TYPEC_TCPM but instead we need  to set DWC3 to gadget-only.